.github/workflows/maintenance-update-version-requirements.yml

```
@@ -0,0 +1,156 @@
+      - name: Install SVN
+        run: sudo apt-get -qq install -y subversion
+
+      - name: Get current WordPress version
+        id: wp-version
+        run: |
+          WP_VERSION=$(svn ls https://core.svn.wordpress.org/tags | sort -V | tail -n 1 | tr -d '/')
+          if [[ -z "$WP_VERSION" ]]; then
+            echo "Failed to get WordPress version"
+            exit 1
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)